cli.js

```diff
@@ -1,5 +1,21 @@
 #!/usr/bin/env node
 
-const useYarn = require('.')
+var fs = require('fs')
+var path = require('path')
+var argv = require('minimist')(process.argv.slice(2));
+var useYarn = require('.')
 
-useYarn()
+var message
+if (argv.f) {
+  message = fs.readFileSync(argv.f, 'utf8')
+}
+
+if (!message && argv.m) {
```

Comment:
Any reason for this instead or `else if(argv.m)`?

Reply:
Nope, that's much better :)